src/System.Private.CoreLib/src/System/Threading/LowLevelThread.cs

```diff
@@ -18,22 +19,16 @@ namespace System.Threading
 {
     internal static class LowLevelThread
```

Comment:
This file is just static methods. RuntimeThread.Windows.cs forwards calls to here, and this calls backs to RuntimeThread. I do not see a clear split in responsibility between the two.

I did not mean to actually change anything functionality wise.